client-sdk/go/crypto/signature/secp256k1/secp256k1.go

```diff
@@ -60,6 +60,5 @@ func (pk PublicKey) Equal(other signature.PublicKey) bool {
 
 // Verify returns true iff the signature is valid for the public key over the context and message.
 func (pk PublicKey) Verify(context, message, signature []byte) bool {
-	// TODO
```

Comment:
?

Reply:
This is obviously still a todo, there were some issues with a linter that were resolved differently so this can be added back.

Reply:
oh thanks